### PR TITLE
Chad/#42 store event description information

### PR DESCRIPTION
**Pull request by chadlagore**
:hourglass: **Status**: _Ready for review_
:tickets: **Ticket(s)**: #42 #41 

## :construction_worker: Changes

+ Storing additional event info, description, url etc.

## :flashlight: Testing Instructions

```
docker-compose test pytest -vvv
```

### CODE REVIEW

File: beluga/models.py [hunk 1]
```diff
@@ -78,9 +78,18 @@ class Event(Base):
 
     start_time = sa.Column(sa.types.DateTime())
     end_time = sa.Column(sa.types.DateTime())
+    start_time_local = sa.Column(sa.types.DateTime())
+    end_time_loal = sa.Column(sa.types.DateTime())
```
Comment (jordanschalm) [inline beluga/models.py]: typo `loal`->`local`

File: beluga/models.py [hunk 2]
```diff
@@ -78,9 +78,18 @@ class Event(Base):
...
+    timezone = sa.Column(sa.String(50))
     location = sa.Column(sa.types.JSON)
     title = sa.Column(sa.String(200))
     attendees = sa.Column(sa.types.JSON)
+    capacity = sa.Column(sa.Integer())
+    logo = sa.Column(sa.types.JSON)
```
Comment (jordanschalm) [inline beluga/models.py]: Logo is... JSON? 🤔

**Comment by jordanschalm** [comment]
Also, I think you'll need to update the events route to this new data out to the client, right? https://github.com/chadlagore/beluga/blob/master/beluga/routes.py#L127

File: beluga/models.py (hunk 1 above)
Comment (jordanschalm) [reply beluga/models.py]: Looks like this is killing the `/events` endpoint

**Comment by chadlagore** [comment]
Indeed I do, next PR.

File: beluga/models.py (hunk 2 above)
Comment (chadlagore) [reply beluga/models.py]: Logo is a little JSON bundle of URL's for different logo sizes. Unless you just want me to pick a size?

File: beluga/models.py (hunk 1 above)
Comment (chadlagore) [reply beluga/models.py]: True, theres more. I'll tinker with that endpoint a bit ☝️

**Comment by chadlagore** [comment]
Update: removed geofiltering on a box, I believe we're going to change the way this works and it was blocking this ticket. The new implementation should use `geoalchemy2`, which will stop us needing to do any in-memory filtering.

**Pull request merged**